CSharpSourceCode/ObjectDataExtensions/CharacterExtendedInfo.cs

```diff
@@ -31,7 +31,7 @@ public class CharacterExtendedInfo
     public class ResistanceTuple
     {
         [XmlAttribute]
-        public DamageType ResistedDamageType = DamageType.Invalid;
+        public DamageType ResistedDamageType = DamageType.Physical;
```

Comment:
I don't agree with this change. The point of setting the default value to Invalid was that we can explicitly see if the value was set from XML or from default instantiation of the class. This breaks that principle. I would like to ask you to change it back or to explain in detail why you think this change is necessary.

Reply:
agree, yeah I was not aware of this thought, for me, it seemed like my damage calculation is taking a value in it, which doesn't need to account for. i will keep then the invalid and will maybe adjust a little the filling of the arrays